README.md

```diff
@@ -154,19 +154,14 @@ properly.
 
 Although it's possible the default configuration would work on your system,
 you'll probably end up wanting to customize it anyhow. There is a default
-`alacritty.yml` at the Git repository root. Alacritty looks for the
-configuration file at the following paths:
+`alacritty.yml` at the Git repository root and on the GitHub's releases page.
+Alacritty looks for the configuration file at the following paths:
```

Comment:
I think I'd rework the entire section a bit and skip out the yml in the repo.

Something like this maybe:

```
You can find the default configuration file with documentation for all available fields on the [GitHub releases page](https://github.com/jwilm/alacritty/releases) for each release.

Alacritty looks for the configuration file at the following paths:
```